packages/fx-core/resource/package.nls.json

```diff
@@ -387,15 +387,18 @@
   "core.common.OutlookWebClientName1": "Outlook web access client id 1",
   "core.common.OutlookWebClientName2": "Outlook web access client id 2",
   "core.common.CancelledMessage": "Operation is cancelled.",
-  "core.common.SpecVersionNotSupported": "OpenAPI version lower than 3.0.0 is not supported.",
-  "core.common.NoServerInformation": "No server information is found in the OpenAPI specification file.",
-  "core.common.MultipleServerInformation": "Multiple server information is found in the OpenAPI specification file.",
+  "core.common.NoServerInformation":  "No server information is found in the OpenAPI specification file.",
   "core.common.RemoteRefNotSupported": "Remote reference is not supported: %s.",
   "core.common.MissingOperationId": "Missing operationIds: %s.",
-  "core.common.NoSupportedApi": "No supported API is found in the OpenAPI specification file.",
+  "core.common.NoSupportedApi": "No supported API is found in the OpenAPI specification file: only GET and POST methods are supported and there can be at most one required parameter and no auth is allowed.",
```

Comment:
confusing about this line: at most one or at least one? no auth is independent of the previous criteria? or it is really AND relation?

Reply:
must GET/POST `and` at most one required parameter  `and`  no auth is allowed. They are AND relation.